specification/columns/skudetails.md

```diff
@@ -0,0 +1,51 @@
+# SKU Details
+
+The **SKU Details** column represents a list of relevant properties shared by all resources having the same [**SKU ID**](#skuid) which are not captured in another FOCUS column. These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics.
```

Comment:
"which are not captured in another FOCUS column."

Can we remove this statement given that we don't want the contents of this column to depend on what other columns are present in FOCUS?

Reply:
Given that it's been confusing to multiple people, I agree that this sentence should be rephrased.